packages/cicero-cli/test/cli.js

```diff
@@ -1041,3 +1092,52 @@ describe('#get', async () => {
     });
 });
 
+describe('#validateVerfiyArgs', () => {
+    it('no args specified', () => {
+        process.chdir(path.resolve(__dirname, 'data/signedArchive/'));
+        const args  = Commands.validateVerifyArgs({
+            _: ['verify']
+        });
+        args.template.should.match(/cicero-cli[/\\]test[/\\]data[/\\]signedArchive$/);
+        args.target.should.match(/ergo/);
+    });
+    it('only target arg specified', () => {
+        process.chdir(path.resolve(__dirname, 'data/signedArchive/'));
+        const args  = Commands.validateVerifyArgs({
+            _: ['verify'],
+            target: 'ergo'
```

Comment:
There is no `target` option for the `verify` command in the CLI. Those tests should be fixed accordingly.